util/tlgen/xbar.sim.core.tpl

```diff
@@ -20,8 +20,21 @@ filesets:
 
 
 targets:
-  sim:
+  sim: &sim_target
```

Comment:
@sriyerg I thought this is the template file for the autogen'd xbar corefiles? or is there another one that needs to be patched?

Reply:
Ohh sorry, nvm..I somehow overlooked this file.